mergify_engine/tasks/engine/__init__.py

```diff
@@ -161,24 +161,73 @@ def run(event_type, data):
         return
 
     with github.get_client(owner, repo, installation) as client:
-        _run(client, event_type, data)
+        pull = get_github_pull_from_event(client, event_type, data)
 
+        if not pull:  # pragma: no cover
+            LOG.info(
+                "No pull request found in the event %s, ignoring",
+                event_type,
+                gh_owner=client.owner,
+                gh_repo=client.repo,
```

Comment:
```suggestion
                gh_owner=owner,
                gh_repo=repo,
```

or is there any reason to access the client values?

Reply:
No that's the same.